test/cmdLineTests/shareClassTests/StoreFilterTests/build.xml

```diff
@@ -83,9 +83,21 @@
 		<copy todir="${build}/Resources">
 			<fileset dir="./Resources/"/>
 		</copy>
-		<javac srcdir="${build}/Resources" destdir="${build}/Resources" debug="true" debuglevel="lines,vars,source" executable="${compiler.javac}" includeAntRuntime="false" encoding="ISO-8859-1" >
-			<src path="${SharedClassUtils_srddir}"/>
-		</javac>
+		<if>
+			<equals arg1="${JAVA_VERSION}" arg2="SE90"/>
+			<then>
+				<property name="addExports" value="--add-modules openj9.sharedclasses --add-exports java.base/com.ibm.oti.util=ALL-UNNAMED" />
+				<javac srcdir="${build}/Resources" destdir="${build}/Resources"  classpath="${build}" fork="true" debug="true" debuglevel="lines,vars,source" executable="${compiler.javac}" includeAntRuntime="false" encoding="ISO-8859-1" >
+					<src path="${SharedClassUtils_srddir}"/>
+					<compilerarg line='${addExports}' />
+				</javac>
+			</then>
+			<else>
+				<javac srcdir="${build}/Resources" destdir="${build}/Resources" fork="true" debug="true" debuglevel="lines,vars,source" executable="${compiler.javac}" includeAntRuntime="false" encoding="ISO-8859-1" >
+					<src path="${SharedClassUtils_srddir}"/>
+				</javac>
+			</else>
+		</if>
 	</target>
```

Comment:
Is this SE90 specific exports? Will the export be needed in the future release? If so, we should have that if statement on SE80?

Reply:
The export will be needed in future releases since they will also have the modules system from Java 9. I can switch the if statement to use SE80 to future proof it.